euler-maven-1/src/main/java/com/ceritoxi/Runner.java

```diff
@@ -0,0 +1,9 @@
+package com.ceritoxi;
+
+public class Runner 
+{
+    public static void main(String[] borsodi) {
```

Comment:
![nofun](https://user-images.githubusercontent.com/2872603/37973228-4e19f584-31da-11e8-86d5-9a6701d977a5.png)

while I get the joke, later on nobody will know wtf is borsodi supposed to mean :D. Use args or arguments plz.